### PR TITLE
Enable molecule tests

### PR DESCRIPTION
**Pull request by joshmoore**
See https://github.com/openmicroscopy/ome-ansible-molecule-dependencies/pull/3

### CODE REVIEW

File: molecule.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  name: galaxy
+  requirements_file: tests/requirements.yml
+driver:
+  name: docker
+docker:
+  containers:
+    - name: ansible-role-analysis-tools
+      image: centos
+      image_version: 7
+      ansible_groups:
```
Comment (manics) [inline molecule.yml]: This isn't used.

File: tests/test_default.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    '.molecule/ansible_inventory').get_hosts('all')
+
+
+def test_hosts_file(File):
```
Comment (manics) [inline tests/test_default.py]: Given how [trivial this role is (installs a single package, should we consider deprecating it?)](https://github.com/joshmoore/ansible-role-analysis-tools/blob/ffd5e16c641f7347c160a59b33d0f73e977136af/tasks/main.yml) you might as well make this a proper test such as:
```
def test_analysis_packages(Package):
    assert Package('parallel')
```

**Review by manics — CHANGES_REQUESTED** [review]

**Comment by joshmoore** [comment]
Pushed & green, @manics

**Comment by manics** [comment]
👍

**Pull request merged**